src/test/java/org/springframework/data/tarantool/repository/BookRepositoryWithTuple.java

```diff
@@ -0,0 +1,12 @@
+package org.springframework.data.tarantool.repository;
+
+import org.springframework.data.tarantool.core.mapping.Tuple;
+import org.springframework.data.tarantool.entities.BookWithoutTupleAnnotation;
+
+
+/**
+ * @author Oleg Kuznetsov
+ */
+@Tuple("test_space")
+public interface BookRepositoryWithTuple extends TarantoolRepository<BookWithoutTupleAnnotation, Integer> {
```

Comment:
Please add custom method without @Tuple to this repository.
TC must call the method and check that it returns an object.